src/Implementations/LinkedLists/cycle_detection.go

```diff
@@ -1,29 +1,25 @@
 package LinkedLists
 
-/*
-Problem statement: Determine if a cycle exists in the given list of nodes
-*/
+// Problem statement: Determine if a cycle exists in the given list of nodes
 
-/*
-Solution 1: Marking nodes
-One way of solving this problem will be marking the nodes as we visit them.
-If a node is marked and visit the same node, we can conclude that a cycle
-exists in the list. But this will require us to change the existing value.
-We mark the value as the negation of the existing value. For example we have
-three nodes
-
-			1 --> 2 --> 3 --> 4 --> 5
-                             /|\	|
-                              -------
-
-As we visit the first node we change it to -1, then we move to the next node.
-If there is a cycle we will definitely visit the marked nodes. If not the loop
-traversing the nodes will terminate. We need to undo the negations before
-returning the function.
-
-Time Complexity: O(n)	Space Complexity: O(1)
-Note: This method will only work if all the values in the list are either positive/negative.
-*/
+// Solution 1: Marking nodes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported function HasCycle1 should be of the form "HasCycle1 ..."](https://app.codacy.com/manual/bumblebee211196/go-ds-daa/pullRequest?prid=5709817)